src/app/features/asset-list/stacks/sip10-token-asset-list/sip10-token-asset-list.tsx

```diff
@@ -24,15 +36,20 @@ export function Sip10TokenAssetList({
     <Stack>
       {tokens.map(token => (
         <Sip10TokenAssetItem
-          balance={token.balance}
+          balance={showBalance ? token.balance : null}
```

Comment:
Is the optional balance `showBalance` and `renderRightElement` so we can re-use `assetItem` to show a toggle or not?

Maybe it would be cleaner to have a new component just for showing the token name and the toggle? 
![Screenshot 2024-07-04 at 13 42 46](https://github.com/leather-io/extension/assets/2938440/3d9bdeb7-a7ae-42c8-985f-d5f8cadc476f)

Reply:
Current implementation was meant to be flexible; any right element, not just a toggle, could be used easily in the future instead of a balance. But you have a valid point of how having just a toggle element and a balance element is cleaner. Will do.

@pete-watters